### ISSUE TITLE
noSmooth() not working on FX2D

### ISSUE THREAD

**Issue by SpyceTewan** [processing/processing#5704]
## Description
noSmooth() does not disable image smoothing in FX2D

## Expected Behavior
Expected to disable antialiasing on images 

## Current Behavior
Antialiasing is still enabled

## Steps to Reproduce
1. Make sketch using FX2D
2. Add noSmooth() to settings()
3. Draw image()

```java
@Override
	public void settings() {
		size(500, 500, FX2D); 
		noSmooth();  // Switching these around does not change anything.
	}
	
	@Override
	public void setup() {
                  // Putting noSmooth() here does not solve it either
	}
	
	@Override
	public void draw() {
		background(0);
		drawGrid();
		image(textures.get(Editor.currentTexture), x, y, x + 200, y + 200); // Image is 16 x 16
	}
```

## Your Environment
*  Processing >3.4
* Windows 10
* Other information:

## Possible Causes / Solutions
The FX2D rendering engine breaks noSmooth(). I'm hoping for a fix.

**Comment by benfry** [comment]
I think JavaFX may not support disabling smoothing, but someone would need to look into it.

**Comment by barefists** [comment]
@benfry JavaFX 8 does not support disabling smoothing. The issue has been [fixed recently](https://bugs.openjdk.java.net/browse/JDK-8204060) in OpenJFX 12, but I'm not sure if Processing supports upgrading to that version.

**Comment by benfry** [comment]
Thanks for the info, @barefists.

**Comment by StefanH-AT** [comment]
Thanks for that!

**Comment by StefanH-AT** [comment]
I don't think this is an issue regarding JavaFX. Lines and other primitive shapes don't get smoothed. This issue only regards images.
